Makefile

```diff
@@ -221,7 +221,6 @@ wasm-rego-testgen-install:
 
 CI_GOLANG_DOCKER_MAKE := $(DOCKER) run \
 	$(DOCKER_FLAGS) \
-	-u $(shell id -u):$(shell id -g) \
```

Comment:
I believe we were doing this to avoid producing binaries owned by root in the working copy. Why remove this?

Reply:
It prohibited us from installing the gcc packages for windows. We could sidestep this by pre-building builder images. 🤔 It's just introducing an extra step the next time we bump golang's version.

Reply:
Well we could give them the same treatment we use for our wasm-builder images.

Reply:
https://github.com/open-policy-agent/opa/runs/2457378935?check_suite_focus=true#step:4:7
```
-rw-r--r--  1 runner docker 49240064 Apr 28 12:14 opa_darwin_amd64
-rw-r--r--  1 runner docker 51711696 Apr 28 12:14 opa_linux_amd64
-rw-r--r--  1 runner docker 64830573 Apr 28 12:14 opa_windows_amd64.exe
```
Looks OK to me. I guess shovelling them through the artifact machinery does away with the root ownership.

Reply:
We're using these flags for `make image` though and that'll produce a binary under the working copy that's owned by root:

```
$ ll ./_release/0.29.0-dev/
total 51M
-rwxr-xr-x 1 root root 51M Apr 28 09:09 opa_docker_amd64
```

Reply:
Ah I wasn't aware. I'll take that into account.:smiley:

Reply:
✔️ This should be done now. 😃